tests/acceptance/reminder-list-test.js

```diff
@@ -35,17 +35,28 @@ test('add new reminder to the page', function(assert) {
 
   visit('/reminders');
   andThen(function() {
-    assert.equal(Ember.$('.reminder-item').length, 5)
+    assert.equal(Ember.$('.reminder-item').length, 5);
   });
   click('.new-button');
 
-  fillIn('.title-input', 'This is a title')
-  fillIn('.reminder-input', 'This is a reminder')
+  fillIn('.title-input', 'This is a title');
+  fillIn('.reminder-input', 'This is a reminder');
 
-  click('.submit-btn')
+  click('.submit-btn');
 
   andThen(function() {
-    assert.equal(Ember.$('.reminder-item').length, 6)
-    assert.equal(Ember.$('.reminder-item:last').text(), 'This is a title')
+    assert.equal(Ember.$('.reminder-item').length, 6);
```

Comment:
Talk to me about why you're using `Ember.` here instead of just `find('.reminder-item'`